charts/misc-apps/values.yaml

```diff
@@ -117,3 +117,25 @@ prometheusMsteams:
   # -- Helm values
   # @default -- [upstream values](https://github.com/prometheus-msteams/prometheus-msteams/blob/master/chart/prometheus-msteams/values.yaml)
   values: {}
+
+# -- [Argo CD Declarative Setup](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/)
+# @default -- [example](./examples/argocd-apps.yaml)
+argocdApps:
+  # -- Enable Argo CD Apps
+  enabled: false
+  name: argocd-apps
+  # -- Annotations for Argo CD Apps app
+  annotations: {}
+  destination:
+    # -- Namespace
+    namespace: infra-argocd
+  # -- Repo URL
+  # @default -- [repo](https://argoproj.github.io/argo-helm)
+  repoURL: https://argoproj.github.io/argo-helm
+  # -- Chart
+  chart: argocd-apps
+  # -- [argocd-apps Helm chart](https://github.com/argoproj/argo-helm/tree/main/charts/argocd-apps) version
+  targetRevision: 5.4.*
```

Comment:
Should have been `0.0.*` since it's a new chart and doesn't match the argocd charts version.